wmfdata/spark.py

```diff
@@ -43,31 +62,38 @@ def start_session_timeout(session):
     session_timeouts[application_id] = timeout
     timeout.start()
 
-def get_session(master='yarn', app_name='wmfdata', spark_config={}):
+def get_session(type="regular", app_name="wmfdata", extra_settings={}):
     """
     Returns an existent Spark session, or a new one if one hasn't yet been created.
     
     Code calling this is responsible for starting a timeout using `start_session_timeout` when it finishes using the session, in order to prevent idle Spark sessions from wasting cluster resources. This function takes cares of cancelling the timeout if the session is returned again before the timeout finishes.
```

Comment:
Comments should be 80 or 120 characters long

Reply:
Ha, yeah, that's pretty nasty. I reformatted all the comments and docstrings in hive.py and spark.py to limit them to 80 characters. I'll do the other files at some point too.